code/measures.parsers

```diff
@@ -1622,6 +1622,12 @@ hasStructsParser
  string pseudoExample struct pldbFile { int rank; char *title; };
  string reference https://en.wikipedia.org/wiki/Struct_(C_programming_language)
 
+hasStructuralTypingParser
+ extends abstractFeatureParser
+ description Does the language feature (significant) structural typing?
+ string title Structural Typing
+ string reference https://en.wikipedia.org/wiki/Structural_type_system
```

Comment:
Interesting! Didnt know about this distinction, compared to duck typing.

Reply:
[PEP 544](https://peps.python.org/pep-0544/) which introduced protocols to Python calls it "static duck typing" :grin: That's a good summary, I think.